README.md

```diff
@@ -45,6 +45,20 @@ It currently bridges PrivacyLx IRC <-> Matrix internal chat rooms. It can be
 extended to support more networks and bridges can be added by configuring
 `matterbridge/templates/matterbridge.toml.j2`.
 
+### Updates
+
+This role deploys unattended updates and updates all system packages in all hosts
```

Comment:
I think we should mention here that only debian templates will be upgraded, if case we ever run another distro. What do you think?

Reply:
Currently we do have only Debian based hosts, we can update the role to support more distributions when/if it's required.